src/core/EdFi.DataManagementService.Core.External/Backend/IDeleteRequest.cs

```diff
@@ -19,7 +19,7 @@ public interface IDeleteRequest
     /// <summary>
     /// The ResourceInfo for the resource being deleted
     /// </summary>
-    IResourceInfo ResourceInfo { get; }
+    ResourceInfo ResourceInfo { get; }
```

Comment:
All this backend work led me to undo some of these trivial interfaces in `External`. They weren't providing anything beyond just moving the underlying record objects to `External`, and direct access seemed reasonable for a backend.